pygotham/frontend/sponsors.py

```diff
@@ -73,7 +73,7 @@ def edit(pk):
     return render_template('sponsors/edit.html', form=form, sponsor=sponsor)
 
 
-@route(blueprint, '/')
+@route(blueprint, '/', navbar_kwargs={'path': ('Sponsors', 'Sponsors')})
```

Comment:
I wasn't saying that we should do it as part of this. I was just asking if Flask-Copilot allowed us to do so. I'm guessing that's where `default_href` comes into play, but I just wanted to check.